_includes/footer.html

```diff
@@ -2,7 +2,7 @@
   <div class="inner">
     <div class="site-footer-inside">
       <p class="site-info"> 
-      © US-RSE  •  2021 
+      © US-RSE  •  2021-2022
```

Comment:
Awesome idea! How about this so we don't need to ever manually update it again?
```suggestion
      © US-RSE  •  2021-{{ 'now' | date: "%Y" }}
```